Unable to run application using npm run test-page, showing below error

I am unable to run application using npm run test-page, showing below error 
===============In Windows 7===============
$ npm run test-page

> simplewebrtc@3.0.0 test-page D:\asergis\eg\SimpleWebRTC
> open https://localhost:8443/test/ && stupid-server -s

'open' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! simplewebrtc@3.0.0 test-page: `open https://localhost:8443/test/ && stu           pid-server -s`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the simplewebrtc@3.0.0 test-page script.
npm ERR! This is probably not a problem with npm. There is likely additional log           ging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\pankaj.k\AppData\Roaming\npm-cache\_logs\2017-06-21T08_53_           51_466Z-debug.log
===================In Ubuntu==============
sudo npm run test-page

> simplewebrtc@3.0.0 test-page /var/www/SimpleWebRTC
> open https://localhost:8443/test/ && stupid-server -s

sh: 1: stupid-server: not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! simplewebrtc@3.0.0 test-page: `open https://localhost:8443/test/ && stupid-server -s`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the simplewebrtc@3.0.0 test-page script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/pankaj/.npm/_logs/2017-06-21T09_41_34_158Z-debug.log

--- a/package.json
+++ b/package.json
@@ -40,7 +40,7 @@
     "updateLatest": "./scripts/updateLatest.sh",
     "lint": "jshint src",
     "validate": "npm ls",
-    "test-page": "open https://localhost:8443/test/ && stupid-server -s",
+    "test-page": "echo \"open https://localhost:8443/test/\" && stupid-server -s",
     "test": "node test/selenium/index.js"
   },
   "pre-commit": [

